ext/jenkins/beaker-tests.sh

```diff
@@ -24,19 +24,12 @@ export BEAKER_PRESERVE_HOSTS=onfail
 export PUPPETDB_PACKAGE_REPO_HOST="puppetdb-prerelease.s3.amazonaws.com"
 export PUPPETDB_PACKAGE_REPO_URL="http://puppetdb-prerelease.s3.amazonaws.com/puppetdb/${PACKAGE_BUILD_VERSION}"
 export PUPPETDB_PACKAGE_BUILD_VERSION=$PACKAGE_BUILD_VERSION
-
-# Setup RVM and gemset
-[[ -s "/usr/local/rvm/scripts/rvm" ]] && source /usr/local/rvm/scripts/rvm
-rvm use ruby-2.0.0-p481
-
-# Remove old vendor directory to ensure we have a clean slate
-if [ -d "vendor" ];
-then
-  rm -rf vendor
-fi
-mkdir vendor
-
-bundle install --path=vendor/bundle --without=test --retry=10
+export BEAKER_CONFIG=$LAYOUT
 
 # Now run our tests
 bundle exec rake beaker:acceptance
+exit_code=$?
+
+$WORKSPACE/ext/jenkins/cleanup-s3.sh
```

Comment:
An out-of-band sweep sounds good, but I'm not sure it's worthwhile for QENG-2729, especially since we will be working to move away from ec2 for puppetdb testing. I think the problem with an out-of-band sweep that makes it difficult to implement immediately is the potential for race conditions since out-of-band presumably means we will be removing all artifacts rather than specifically-versioned artifacts as seen here: https://github.com/waynr/puppetdb/blob/qeng-2729/ext/jenkins/packaging-cleanup.sh#L14